paasta_tools/cli/cmds/status.py

```diff
@@ -838,6 +839,21 @@ def print_flink_status(
         output.append(f"    No other information available in non-running state")
         return 0
 
+    pod_running_count = pod_evicted_count = pod_other_count = 0
+    for pod in status.pod_status:
+        if pod["phase"] == "Running":
+            pod_running_count += 1
+        elif pod["phase"] == "Failed" and pod["reason"] == "Evicted":
```

Comment:
Any reason why we don't want to show failed and evicted separately. In verbose mode you are showing failed and evicted as separate. Might be misleading if we see evicted here and failed in verbose description.

Reply:
When a Pod is evicted, `phase` will be 'Failed', and `reason` will be 'Evicted'.
On verbose mode, the list will look like this:
```
    Pods:
      Pod Name      Host                         Phase
      xxxxxx        ip-x-x-x-x..compute.internal Running
      xxxxxx        ip-x-x-x-x.compute.internal  Running
      xxxxxx        ip-x-x-x-x.compute.internal  Failed
        Evicted: Disk quota exceeded
      xxxxxx        ip-x-x-x-x..compute.internal Running
      xxxxxx        ip-x-x-x-x..compute.internal Running
      xxxxxx        ip-x-x-x-x..compute.internal Running
```